docs/design/Profiles.md

```diff
@@ -0,0 +1,53 @@
+# Profiles
+
+## Overview
+
+Profiles is a configuration format and feature of Zonemaster Engine.
+Each of Zonemaster CLI, Backend and GUI allow users and administrators to configure Zonemaster Engine with different profiles.
+
+
+## Background
+
+The Zonemaster requirements and specification documents speak of user-configurable profiles and policies for Zonemaster Engine and CLI.
+
+In release 2017.4 the profile and policy concept from the requirements and specifications were integrated into single data structure, also called a profile.
+
```

Comment:
Suggest to modify the above text as follows:
In release 2017.4 all earlier configurable features such as config,policy, filters and profile concept from the requirements and specifications were integrated into single data structure, also called a profile.